core/templates/dev/head/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -32,13 +32,15 @@ oppia.factory('ExplorationObjectFactory', [
       INTERACTION_DISPLAY_MODE_INLINE,
       INTERACTION_SPECS) {
     var Exploration = function(
-        initStateName, paramChanges, paramSpecs, states, title, languageCode) {
+        initStateName, paramChanges, paramSpecs, states, title, languageCode,
+        imageCounter) {
       this.initStateName = initStateName;
       this.paramChanges = paramChanges;
       this.paramSpecs = paramSpecs;
       this.states = states;
       this.title = title;
       this.languageCode = languageCode;
+      this.image_counter = imageCounter;
```

Comment:
lowerCamel casing*

Reply:
Why this comment isn't resolved until now? is there any issue with this?

Reply:
sorry, for delay.
Done